app/src/main/java/com/hedvig/app/feature/marketing/ui/BackgroundImage.kt

```diff
@@ -0,0 +1,46 @@
+package com.hedvig.app.feature.marketing.ui
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.BoxScope
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.SideEffect
+import androidx.compose.runtime.getValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.layout.ContentScale
+import coil.compose.rememberImagePainter
+import coil.size.Scale
+import com.google.accompanist.systemuicontroller.rememberSystemUiController
+import com.hedvig.app.feature.marketing.Background
+import com.hedvig.app.util.compose.rememberBlurHash
+
+@Composable
+fun BackgroundImage(background: Background, content: @Composable BoxScope.() -> Unit) {
```

Comment:
Is this the API we want to expose for the background image? The way this will work now is that if the background is still loading, or takes a long time to load, or fails to load, the rest of the screen will never draw on the screen.
What if where we fetch the image from fails but we still want to have the rest of the screen to work just fine possibly with a placeholder background or some plain color or whatever.
I'd imagine the background image and the front content to be in the same Box scope, but have the be individually drawing on the screen. What do you think? Sort of one laid out on top of the other.

Reply:
Definitely not - this was a mistake on my part. The intended behaviour was to show the content, regardless of if the background image loads or not.

Reply:
If the background image fails to load, we still have a default background color showing, which I believe should be sufficient.